README.md

```diff
@@ -66,6 +66,8 @@ This will install `http-server` globally so that it may be run from the command
 
 `-U` or `--utc` Use UTC time format in log messages.
 
+`-ip` Enable client IP logging
```

Comment:
Because of POSIX conventions, we should make this `--ip`, otherwise it looks like `-i` and `-p` together.